ctrd/image.go

```diff
@@ -212,11 +203,13 @@ outer:
 				}
 				// update status of active entries!
 				for _, active := range active {
```

Comment:
maybe we should rename the later **active** as activities. It's a little misunderstanding

Reply:
Have updated